inspire_disambiguation/config.py

```diff
@@ -21,20 +21,3 @@
 # or submit itself to any jurisdiction.
 
 """Disambiguation configuration."""
```

Comment:
what is this file used for now?

Reply:
You can put config variables there as things in app.py shouldn't be touched.